alot/crypto.py

```diff
@@ -49,7 +19,7 @@ def RFC3156_micalg_from_algo(hash_algo):
     :rtype: str
     """
     # hash_algo will be something like SHA256, but we need pgp-sha256.
-    hash_algo = _hash_algo_name(hash_algo)
+    hash_algo = gpg._gpgme.gpgme_hash_algo_name(hash_algo)
```

Comment:
Sure.  The reason is that no one has written a wrapper.  We are a free software project, and welcome contributions.